products/nextjs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Next.js
+category: framework
+tags: javascript-runtime
+iconSlug: nextdotjs
+permalink: /nextjs
+alternate_urls:
+-   /next-js
+-   /nextdotjs
+versionCommand: npx next --version
+releasePolicyLink: https://github.com/vercel/next.js/blob/canary/contributing/repository/release-channels-publishing.md
```

Comment:
```suggestion
```

Unfortunately this link does not contain detailed information about the release cadence or release / EOL policy, so it's better to remove it. I have read https://nextjs-forum.com/post/1140967772285849660 and it seems only the latest release is supported.